### ISSUE TITLE
Angular 10: Dev Server Hangs on Load or Bundle Instantiation.

### ISSUE THREAD

**Issue by toneware** [stackblitz/core#1306]
I have a paid subscription to Stackblitz, so I created a practice Angular Project on Chrome (https://stackblitz.com/edit/macs); it was working swimmingly (for several days) until I let it create a GitHub repo. Then Stackblitz started hanging on the "loading dev server" or "instantiating bundle" phase. 

After trying to resolve the issue with a clever combination of reboots and ctrl-f5s, I totally deleted the git hub repo and the stackblitz project, created a new project, and am now seeing the same issue. FWIW, it ran correctly the first time when I created the new project. I added one line to a component:
this.router.navigateByUrl("/");
I didn't like the results, so I commented that line out. That's when Stackblitz started locking up again.

**This is a show-stopper for me.**

![2020-09-23 11_39_46-macs - StackBlitz](https://user-images.githubusercontent.com/16153437/94042790-9e87b300-fd91-11ea-979b-3a6f9558f02a.png)
![2020-09-23 11_40_13-macs - StackBlitz](https://user-images.githubusercontent.com/16153437/94042799-a0ea0d00-fd91-11ea-9181-b96125c35c71.png)

![2020-09-23 12_07_08-Settings - About Chrome](https://user-images.githubusercontent.com/16153437/94045653-59fe1680-fd95-11ea-868c-bfe0286fa28b.png)

**Comment by toneware** [comment]
Just tried it in incognito mode, to no avail.

BTW, cookies are enabled for stackblitz:

![2020-09-23 12_02_07-Settings - Cookies and other site data](https://user-images.githubusercontent.com/16153437/94045296-e4924600-fd94-11ea-9326-76bb61360437.png)

**Comment by EricSimons** [comment]
Hey! I just checked this out and I'm seeing this error:

![Screen Shot 2020-09-23 at 10 30 11 AM](https://user-images.githubusercontent.com/556934/94048096-d689f880-fd87-11ea-872c-d12d3fb86ade.png)

It seems like there might be a bug in your code that's causing that angular app to crash and go into an infinite redirect loop?

**Comment by toneware** [comment]
Eric,

Thanks for the quick response!

This is very interesting. I have not seen that error pop up before (still don't). Even so, I put in a little code to return if the object is null, just in case. When I try running that, I get the same, exact errors as before. 

Even more interesting is that this part of the code hasn't changed in days. The error that you pointed out has never appeared before. Even so, while you may be totally correct, but I can't get past the monaco-compiler errors. 

Actually, though, I am seeing something new: 
![2020-09-23 12_50_12-macs - StackBlitz](https://user-images.githubusercontent.com/16153437/94050122-71d89900-fd9b-11ea-8e83-836885ff7f7b.png)

**Comment by toneware** [comment]
Why is it that you can see that error, while I can't get the dev server to finish the build?

**Comment by EricSimons** [comment]
The monaco errors are actually just warnings that sourcemaps are missing- we're putting out a fix shortly which should make those go away!

Interestingly, it looks like the build is now working (albeit with some more errors in the console originating from your application code):
![Screen Shot 2020-09-23 at 2 55 12 PM](https://user-images.githubusercontent.com/556934/94073784-e49e4000-fdac-11ea-8bcd-4e1de8fc9cf4.png)

I'm wondering if there was a bad cache on your machine or something? Perhaps try clearing cache and trying again? Let me know if this is still an issue after that. Incredibly weird 😅

**Comment by toneware** [comment]
Thanks again, Eric. 

So, I managed to isolate the issue -- it was a route that redirected to a missing component. That is, I removed the landing page component, and the default "/" route had no where to redirect to. Sounds a lot like your initial hunch.  Anyhow, I am able to move forward, but I still don't know why I couldn't get past the dev server.

So, I'll close this ticket with this comment.

Thanks again for jumping right on this -- I really appreciate it!
